diff --git a/bootstrap/__init__.py b/bootstrap/__init__.py
--- a/bootstrap/__init__.py	2026-06-18 10:10:16.672488 +0000
+++ b/bootstrap/__init__.py	2026-06-25 06:46:42.343092 +0000
@@ -5,7 +5,9 @@
 import json
 from pathlib import Path
 
-SNAPSHOT_PATH = Path(__file__).resolve().parent.parent / 'reference_data' / 'subsystems' / 'bootstrap.json'
+SNAPSHOT_PATH = (
+    Path(__file__).resolve().parent.parent / 'reference_data' / 'subsystems' / 'bootstrap.json'
+)
 _SNAPSHOT = json.loads(SNAPSHOT_PATH.read_text())
 
 ARCHIVE_NAME = _SNAPSHOT['archive_name']
