diff --git a/bridge/exceptions.py b/bridge/exceptions.py
--- a/bridge/exceptions.py	2026-06-18 10:10:15.864693 +0000
+++ b/bridge/exceptions.py	2026-06-22 09:03:22.235071 +0000
@@ -54,9 +54,9 @@
 
     def __repr__(self) -> str:
         return (
-            f'BridgeFatalError(status={self.status}, '
-            f'error_type={self.error_type!r}, message={str(self)!r})'
+            f"BridgeFatalError(status={self.status}, "
+            f"error_type={self.error_type!r}, message={str(self)!r})"
         )
 
 
-__all__ = ['BridgeAuthError', 'BridgeFatalError', 'EpochSupersededError']
+__all__ = ["BridgeAuthError", "BridgeFatalError", "EpochSupersededError"]
