diff --git a/bridge/close_codes.py b/bridge/close_codes.py
--- a/bridge/close_codes.py	2026-06-18 10:10:15.881758 +0000
+++ b/bridge/close_codes.py	2026-06-22 09:03:22.205436 +0000
@@ -44,9 +44,9 @@
 
 
 __all__ = [
-    'WS_CLOSE_EPOCH_MISMATCH',
-    'WS_CLOSE_INIT_FAILURE',
-    'WS_CLOSE_PERMANENT_UNAUTHORIZED',
-    'WS_CLOSE_RECONNECT_BUDGET_EXHAUSTED',
-    'WS_CLOSE_SESSION_NOT_FOUND',
+    "WS_CLOSE_EPOCH_MISMATCH",
+    "WS_CLOSE_INIT_FAILURE",
+    "WS_CLOSE_PERMANENT_UNAUTHORIZED",
+    "WS_CLOSE_RECONNECT_BUDGET_EXHAUSTED",
+    "WS_CLOSE_SESSION_NOT_FOUND",
 ]
