diff --git a/keybindings/__init__.py b/keybindings/__init__.py
--- a/keybindings/__init__.py	2026-06-18 10:10:16.827175 +0000
+++ b/keybindings/__init__.py	2026-06-25 15:09:03.000000 +0800
@@ -14,7 +14,9 @@
 import json
 from pathlib import Path
 
-SNAPSHOT_PATH = Path(__file__).resolve().parent.parent / 'reference_data' / 'subsystems' / 'keybindings.json'
+SNAPSHOT_PATH = (
+    Path(__file__).resolve().parent.parent / 'reference_data' / 'subsystems' / 'keybindings.json'
+)
 _SNAPSHOT = json.loads(SNAPSHOT_PATH.read_text())
 
 ARCHIVE_NAME = _SNAPSHOT['archive_name']
