diff --git a/native_ts/__init__.py b/native_ts/__init__.py
--- a/native_ts/__init__.py	2026-06-18 10:10:16.689041 +0000
+++ b/native_ts/__init__.py	2026-06-25 15:09:03.000000 +0800
@@ -5,7 +5,9 @@
 import json
 from pathlib import Path
 
-SNAPSHOT_PATH = Path(__file__).resolve().parent.parent / 'reference_data' / 'subsystems' / 'native_ts.json'
+SNAPSHOT_PATH = (
+    Path(__file__).resolve().parent.parent / 'reference_data' / 'subsystems' / 'native_ts.json'
+)
 _SNAPSHOT = json.loads(SNAPSHOT_PATH.read_text())
 
 ARCHIVE_NAME = _SNAPSHOT['archive_name']
