diff --git a/types/__init__.py b/types/__init__.py
--- a/types/__init__.py	2026-06-18 10:10:15.651871 +0000
+++ b/types/__init__.py	2026-06-22 09:03:28.138179 +0000
@@ -66,7 +66,9 @@
     stream_event_to_dict,
 )
 
-SNAPSHOT_PATH = Path(__file__).resolve().parent.parent / "reference_data" / "subsystems" / "types.json"
+SNAPSHOT_PATH = (
+    Path(__file__).resolve().parent.parent / "reference_data" / "subsystems" / "types.json"
+)
 _SNAPSHOT = json.loads(SNAPSHOT_PATH.read_text())
 
 ARCHIVE_NAME = _SNAPSHOT["archive_name"]
