diff --git a/bridge/bridge_enabled.py b/bridge/bridge_enabled.py
--- a/bridge/bridge_enabled.py	2026-06-18 10:10:15.916813 +0000
+++ b/bridge/bridge_enabled.py	2026-06-22 09:03:22.166634 +0000
@@ -90,12 +90,12 @@
 
 
 __all__ = [
-    'check_bridge_min_version',
-    'get_bridge_disabled_reason',
-    'get_ccr_auto_connect_default',
-    'is_bridge_enabled',
-    'is_bridge_enabled_blocking',
-    'is_ccr_mirror_enabled',
-    'is_cse_shim_enabled',
-    'is_env_less_bridge_enabled',
+    "check_bridge_min_version",
+    "get_bridge_disabled_reason",
+    "get_ccr_auto_connect_default",
+    "is_bridge_enabled",
+    "is_bridge_enabled_blocking",
+    "is_ccr_mirror_enabled",
+    "is_cse_shim_enabled",
+    "is_env_less_bridge_enabled",
 ]
